packages/flutter/test/cupertino/scrollbar_test.dart

```diff
@@ -164,6 +164,58 @@ void main() {
     await tester.pump(_kScrollbarFadeDuration);
   });
 
+  testWidgets('When isAlwaysShown is truew, must pass a controller',
+      (WidgetTester tester) async {
+    Widget viewWithScroll() {
+      return Directionality(
+        textDirection: TextDirection.ltr,
+        child: MediaQuery(
+          data: const MediaQueryData(),
+          child: CupertinoScrollbar(
+            isAlwaysShown: true,
+            child: const SingleChildScrollView(
+              child: SizedBox(
+                width: 4000.0,
+                height: 4000.0,
+              ),
+            ),
+          ),
+        ),
+      );
+    }
+
+    expect(() async {
+      await tester.pumpWidget(viewWithScroll());
```

Comment:
Will this test pass prior to the change? Before this change the assert should still trigger at the end of the frame right?

Reply:
I just tried it and it fails prior to the change (so does the corresponding material test).

It does still throw an error though, but I guess it's happening in the post frame callback and so it's not caught by this.